src/main/java/duke/ui/map/LocationCard.java

```diff
@@ -0,0 +1,25 @@
+package duke.ui.map;
+
+import duke.model.locations.Venue;
+import duke.ui.UiPart;
+import javafx.scene.control.Tooltip;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.StackPane;
+
+public class LocationCard extends UiPart<StackPane> {
+    private static final String FXML = "LocationCard.fxml";
+
+    private LocationCard(Venue location) {
+        super(FXML);
+        double offsetY = 600 - ((location.getLatitude() - 1.218) * 600 / (1.486 - 1.218));
```

Comment:
can you make this size agnostic? things many not appear the same way on all resolution screens